.github/pull_request_template.md

```diff
@@ -25,5 +25,6 @@ Otherwise, please check the following TODO list:
 - [ ] Replace `<COPYRIGHT HOLDER>` in `LICENSE` of your package with your name
 - [ ] Fill out `README.md` in your package
 - [ ] Add import statements of your function or class names to be used in `__init__.py`
+- [ ] (Optional) Add `from __future__ import annotations` at the head of any Python files that include MyPy typing to support older Python versions
```

Comment:
How about dropping `MyPy`? `from __future__ import annotations` is not just for MyPy.

```suggestion
- [ ] (Optional) Add `from __future__ import annotations` at the head of any Python files that include typing to support older Python versions
```

Reply:
Sorry for the late response, it completely slipped my mind!
Yes, I agree with you and I will remove it!